src/src/AppBundle/Resources/views/confirmedRegistration.html.twig

```diff
@@ -1 +0,0 @@
-<p> You have successfully confirmed your registration.</p>
```

Comment:
Preco sa toto zmazalo?

Reply:
osobne si myslim, ze netreba posielat confirmation pre confirmation. Takze za mna je to ok.

Reply:
lebo ja som to nazaciatku spravila takto, lenze potom to asi Zoli prerobil cez templaty a aby tam boli aj jazyky. A toto sa tym padom uz nevyuziva.